blogposts/2021/workspaces-of-sourcegraph.md

```diff
@@ -0,0 +1,224 @@
+---
+title: The workspaces of Sourcegraph
```

Comment:
We should consider a broader title (I borrowed this title to intro the workspaces section). From my perspective, the essence of the post is about how developers prefer to customize/personalize their workspaces. Here are some ideas:

- What we found after analyzing X# of developer workspaces 
- Why developers prefer to personalize their dev workspaces (needs to answer this question in detail in the intro) 
- How to improve the display of code (needs to answer this question in detail - may be a follow up post) 
- Analyzing the workspaces of Sourcegraph 
- What a developer's workspace says about their workflow 
- What makes developer workspaces unique?

Reply:
When I read this title I thought it was about home office setups